### DISCUSSION TITLE
Does orientDB support TLS v1.3?

### DISCUSSION THREAD

**Discussion by makandre** [orientechnologies/orientdb#10191] [Q&A] [answered]
As the title says, is TLS v1.3 supported?  https://orientdb.org/docs/3.1.x/security/Using-SSL-with-OrientDB.html This page only says TLS v1.2.

If TLS v1.3 is support, how to enable it?

**Comment by tglman** [comment, marked as answer]
Hi, 

OrientDB use the jdk built in ssl support, so it depends on the jdk you are running on.

**Reply by makandre** [reply]
And if I'm using the docker image https://hub.docker.com/_/orientdb/, that's seems to be based on eclipse-temurin:8-jdk, which supports 1.3 as far as I can tell.

Is my understanding correct?

**Reply by tglman** [reply]
yes
